lib/mixins/validatable-http-message.js

```diff
@@ -275,22 +270,19 @@ not a parseable JSON:\n${error.message}\
           jph.parse(this.expected.body);
           this.validation.body.expectedType = expectedContentType;
         } catch (error) {
-          message = {
+          this.validation.body.results.push({
             message: `\
 Can't validate. Expected body 'Content-Type' is '${expectedContentType}' \
 but body is not a parseable JSON:\n${error.message}\
 `,
             severity: 'error'
-          };
-
-          this.validation.body.results.push(message);
+          });
         }
       } else {
         try {
           jph.parse(this.expected.body);
           this.validation.body.expectedType = 'application/json';
-        } catch (error3) {
```

Comment:
lol, error3

Reply:
Nasty variables shadowing after decaffeination. I've rewritten this block to be more sane, but more profound changes will be introduced under #150.